### PR TITLE
Update deploy-MatrixOne-cluster.md

### PR DESCRIPTION
**Pull request by yanghaitao5000**
Updated operator installation method

## What type of PR is this?

- [ ] Enhancement
- [ ] Displaying
- [ ] Typo
- [ ] Doc Request

## Which issue(s) this PR fixes:
https://github.com/matrixorigin/matrixorigin.io.cn/issues/717
https://github.com/matrixorigin/matrixorigin.io.cn/issues/747

issue #

## What this PR does / why we need it:
Updated operator deployment method

### CODE REVIEW

**Comment by dengn** [comment]
please run the following to get lint passed:

安装依赖：npm ci
校验：npm run lint
校验并自动修复：npm run lint:fix

File: scripts/mdast-words-count/mdast-words-count
Comment (dengn) [inline scripts/mdast-words-count/mdast-words-count:1]: please remove these scripts files, they are unrelated.

**Review by dengn — CHANGES_REQUESTED** [review]

**Comment by yanghaitao5000** [comment]
该问题已在: https://github.com/matrixorigin/matrixorigin.io.cn/pull/794 解决;

**Comment by yanghaitao5000** [comment]
> Updated operator installation method
> 
> ## What type of PR is this?
> * [ ]  Enhancement
> * [ ]  Displaying
> * [ ]  Typo
> * [ ]  Doc Request
> 
> ## Which issue(s) this PR fixes:
> #717 #747
> 
> issue #
> 
> ## What this PR does / why we need it:
> Updated operator deployment method



> Updated operator installation method
> 
> ## What type of PR is this?
> * [ ]  Enhancement
> * [ ]  Displaying
> * [ ]  Typo
> * [ ]  Doc Request
> 
> ## Which issue(s) this PR fixes:
> #717 #747
> 
> issue #
> 
> ## What this PR does / why we need it:
> Updated operator deployment method

**Pull request closed without merging**